recipes/pytest-pyvista/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "pytest-pyvista" %}
+{% set version = "0.1.8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/p/pytest_pyvista/pytest_pyvista-{{ version }}.tar.gz
+  sha256: 43e6f0a85b6ff23aff51b7f750528de0fdda6da2011a94b636f05d8498fe38a0
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.8
+  run:
```

Comment:
Does this list come from pytest-pyvista[tests]?  Only pytest (and pyvista) is necessary to run this package.  I believe pyvista was removed due to problems with vtk installation (vtk vs. vtk-osmesa and so on).  I'm not familiar enough with conda to comment on whether it should go in here.

Reply:
`pytest-pyvista[tests]` is what is used for testing `pytest_pyvista` itself.

Reply:
Thank you. I will only use `pytest` and `pyvista` as dependencies, I looked at `geovista`'s feedstock and it looks fine to have `pyvista` as a dependency.

Reply:
See 92cd072d9da658761fde4790fdbc7dbd4aa9e4c6 .